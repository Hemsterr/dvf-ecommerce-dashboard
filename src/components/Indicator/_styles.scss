.indicator {
  @include absolute-position(0, auto, auto, 0);

  width: 100%;
  height: 100%;
  z-index: 1;
  opacity: 0.5;
  background-color: $black;

  &__img {
    @include absolute-position(50%, auto, auto, 50%);

    width: pxToRem(25px);
    height: pxToRem(25px);
    transform: translate(-50%, -50%);
    border: pxToRem(5px) solid $white;
    animation: spin 2s linear infinite;
    display: inline-block;

    border: {
      top: pxToRem(5px) solid $gray;
      radius: 50%;
    }
  }
}

@keyframes spin {
  0% { transform: rotate(0deg); }
  100% { transform: rotate(360deg); }
}
