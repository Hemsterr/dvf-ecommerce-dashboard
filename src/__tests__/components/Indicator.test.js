// Libs
import React from 'react'
import { shallowToJson } from 'enzyme-to-json'
import { shallow } from 'enzyme'

// Component
import Indicator from '../../components/Indicator'

describe('Indicator component', () => {
  const wrapper = shallow(<Indicator />)

  // Snapshot
  test('should render correctly Indicator component', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
