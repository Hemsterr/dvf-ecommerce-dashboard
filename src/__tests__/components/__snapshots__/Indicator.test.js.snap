// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Indicator component should render correctly Indicator component 1`] = `
<div
  className="indicator"
>
  <span
    className="indicator__img"
  />
</div>
`;
